### PR TITLE
Add components in formpage and create skeleton of formpage

### LINKED ISSUE #113: Create FormTextbox Component

### LINKED ISSUE #169: Create an outline of the form page

### PR DESCRIPTION
**Pull request by sonia-y**
Fix #113 
Fix #169

### CODE REVIEW

**Comment by warrenshen** [comment]
@sonia-y your branch has merge conflicts!!!! :worried:

File: app/assets/javascripts/components/form/form_body.es6.jsx [hunk 1]
```diff
@@ -8,25 +8,36 @@ class FormBody extends Component {
...
       },
-    };
+    }
+  }
 
   render() {
     return(
-      <div style={this.styles.container}>
-        <div style={this.styles.title}>{'Form'}</div>
+
```
Comment (warrenshen) [inline app/assets/javascripts/components/form/form_body.es6.jsx]: Is there an extra space here?

File: app/assets/javascripts/components/form/form_section.es6.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      },
+      line: {
+        flex: '1',
+        height: '1px',
+        marginLeft: '10px',
+        borderTop: 'solid gray 1px',
+      },
+      infoContainer: {
+        padding: '16px',
+      }
```
Comment (warrenshen) [inline app/assets/javascripts/components/form/form_section.es6.jsx]: Missing a trailing comma here!

File: app/assets/javascripts/components/form/form_body.es6.jsx (hunk 1 above)
Comment (maxwolffe) [inline app/assets/javascripts/components/form/form_body.es6.jsx]: extra newline here?

File: app/assets/javascripts/components/form/form_section.es6.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    return(
+      <div style={this.styles.container}> 
+        <div style={this.styles.titleContainer}> 
+          <span style={this.styles.title}>{this.props.title}</span>
+          <span style={this.styles.line} /> 
+        </div>
+        <div style={this.styles.infoContainer}>
+          <FormTextbox 
+            title={'First Name'}
+            placeHolder={'Emily'}/>
```
Comment (warrenshen) [inline app/assets/javascripts/components/form/form_section.es6.jsx]: Don't forget your space before the `/>`!

File: app/assets/javascripts/components/form/form_textbox.es6.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        fontSize: StyleConstants.fonts.sizes.smaller,
+      },
+      input: {
+        alignSelf: 'flex-end',
+        width: '250px',
+        height: '30px',
+        padding: '4px',
+        fontSize: StyleConstants.fonts.sizes.smallest, 
+      },
+    }
```
Comment (warrenshen) [inline app/assets/javascripts/components/form/form_textbox.es6.jsx]: Don't forget a semicolon here!

File: app/assets/javascripts/components/form/form_section.es6.jsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+class FormSection extends Component { 
+
+//TODO(Sonia): Add a 'required' proptype
+  static get PropTypes() {
```
Comment (unzunz) [inline app/assets/javascripts/components/form/form_section.es6.jsx]: `static get propTypes() {`

**Comment by maxwolffe** [comment]
LGTM

File: app/assets/javascripts/components/form/form_section.es6.jsx (hunk 2 above)
Comment (unzunz) [inline app/assets/javascripts/components/form/form_section.es6.jsx]: trailing comma

File: app/assets/javascripts/components/form/form_textbox.es6.jsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+class FormTextbox extends Component { 
+
+  static get PropTypes() {
```
Comment (unzunz) [inline app/assets/javascripts/components/form/form_textbox.es6.jsx]: `static get propTypes() {`

**Pull request merged**